.github/workflows/cypress.yml

```diff
@@ -0,0 +1,35 @@
+name: Cypress Tests
+
+on: [push]
+
+jobs:
+
+  test-material-ui-viewer:
+    name: ITK Viewer Material UI Tests
+    runs-on: ubuntu-20.04
+    container:
+      image: cypress/browsers:node16.14.0-chrome99-ff97
+      options: --user 1001
```

Comment:
In order to get the tests to work on Firefox we need to use non-root user (see [docs](https://github.com/cypress-io/github-action#firefox))